examples/core/gated_gcn/train.py

```diff
@@ -123,20 +150,21 @@ def main():
 
     opt = optim.Adam(model.parameters(), lr=0.01)
     loss_fn = nn.BCEWithLogitsLoss()
+    acc = evaluate(model, device, test_loader, evaluator)
 
     print("---------- Training ----------")
-    for epoch in range(50):
+    for epoch in range(args.num_epochs):
         # Kick off training.
+        t0 = time.time()
         loss = train(model, device, train_loader, opt, loss_fn)
-
+        t1 = time.time()
         # Evaluate the prediction.
-        valid_acc = evaluate(model, device, valid_loader, evaluator)
-        test_acc = evaluate(model, device, test_loader, evaluator)
+        evaluate(model, device, valid_loader, evaluator)
+        acc = evaluate(model, device, test_loader, evaluator)
```

Comment:
for test evaluate, we could just evaluate once, just after the train.

Reply:
OK, I will change the `Accuracy` during training as validate accuracy.